src/Symfony/Component/RateLimiter/FixedWindowLimiter.php

```diff
@@ -33,49 +34,72 @@ final class FixedWindowLimiter implements LimiterInterface
 
     public function __construct(string $id, int $limit, \DateInterval $interval, StorageInterface $storage, ?LockInterface $lock = null)
     {
+        if ($limit < 1) {
+            throw new \InvalidArgumentException(sprintf('Cannot set the limit of "%s" to 0, as that would never accept any hit.', TokenBucketLimiter::class));
```

Comment:
```suggestion
            throw new \InvalidArgumentException(sprintf('Cannot set the limit of "%s" to 0, as that would never accept any hit.', __CLASS__));
```

Wrong class: This is the constructor of `FixedWindowLimiter`.